### DISCUSSION TITLE
Command to open external documentation for item under cursor

### DISCUSSION THREAD

**Discussion by antoyo** [helix-editor/helix#7448] [Ideas]
#### Describe your feature request

I'd like a command to open the external documentation for an item under the cursor, like [this](https://www.reddit.com/r/rust/comments/w7cf7d/rustanalyzer_tip_open_docsrs_for_the_symbol_under/).
For instance, it will open docs.rs for a symbol in a Rust buffer.

**Comment by AlexanderBrevig** [comment]
Seem like this is a [rust-analyzer LSP Extension](https://github.com/rust-lang/rust-analyzer/blob/master/docs/dev/lsp-extensions.md#open-external-documentation) and as such would only work for `rust` lang. I started looking into implementing this, but got a bit unsure of the best way forward as it's not part of the current LSP spec.

Ideas?

**Comment by the-mikedavis** [comment]
There was some prior discussion[^1] about language-specific LSP extensions. It would be perfect for a language-specific plugin once the plugin-system is featureful enough to support it, but it's a bad fit for core since it's specific to language or small subset of languages. For now it's best to keep any necessary changes up to date in a fork

[^1]: I can't find the links to the issues/discussions/matrix-chats but I think this has come up for non-standard typescript/deno location types and some omnisharp and solargraphs messages. I'll update this comment if I can find them
